go/inst/instance_topology.go

```diff
@@ -553,25 +553,55 @@ Cleanup:
 	return instance, err
 }
 
-func canMoveViaGTID(instance, otherInstance *Instance) (isOracleGTID bool, isMariaDBGTID, canMove bool) {
-	isOracleGTID = (instance.UsingOracleGTID && otherInstance.SupportsOracleGTID)
-	isMariaDBGTID = (instance.UsingMariaDBGTID && otherInstance.IsMariaDB())
+func canReplicateAssumingOracleGTID(instance, masterInstance *Instance) (canReplicate bool, err error) {
+	instanceTotalGtid := fmt.Sprintf("%s,%s", instance.GtidPurged, instance.ExecutedGtidSet)
```

Comment:
Since `gtid_executed` is a superset of `gtid_purged`, I believe you can just use instance.ExecutedGtidSet as-is for this. (But I'm not too familiar with Orchestrator's codebase, so apologies if I'm misreading!)

Reply:
Thank you!